### DISCUSSION TITLE
Should we release 0.71.1?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#50] [Patches]
### Released, please refer to https://github.com/reactwg/react-native-releases/discussions/53

---
Current Release: [0.71.0](https://github.com/facebook/react-native/releases/tag/v0.71.0)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

- https://github.com/facebook/react-native/commit/64475aeb3b1b0e37bdad2c2d5abb2116b2141c06
- https://github.com/facebook/react-native/commit/5eb25d2186fd94128ec2ae74fec02429da969e63
- https://github.com/facebook/react-native/commit/f47b5b8b5def41aeb6d5b672928cc57e20fba49d
- https://github.com/facebook/react-native/commit/e948c79bda7f000427d56dac6fe5c70555db5701
- https://github.com/facebook/react-native/commit/a80cf96fc8821dcc07987c9b91dff0c839c8769c
- https://github.com/facebook/react-native/commit/38e35df47c59e425fdac34d1f91540b8b8ca2908
- https://github.com/facebook/react-native/commit/732a02a3c8eef0a9f01f58574aa96f2b47518566
- https://github.com/facebook/react-native/commit/0c53420a7af306d629350e1244e8e2ccae08a312
- https://github.com/facebook/react-native/commit/b314e6f147fdfcd71e7ba3b17338bb44356df98e
- https://github.com/facebook/react-native/pull/35847
- https://github.com/facebook/react-native/pull/35786
- bump CLI to 10.1.2 to get
  - https://github.com/react-native-community/cli/pull/1798
  - https://github.com/react-native-community/cli/pull/1800
- https://github.com/facebook/react-native/commit/470f79b61734e085978b03507eda6ed026936b7b
- https://github.com/facebook/react-native/commit/477f546d1b7630109e40ec0e6084fbcfa3a3e285
- https://github.com/facebook/react-native/commit/60b9d8c2b9800135fdace5f58d94bebde2849510
- https://github.com/facebook/react-native/pull/35869

#### Issues that are top of mind

- https://github.com/facebook/react-native/issues/35838
  - fix in the works: https://github.com/react-native-community/cli/pull/1798
- https://github.com/facebook/react-native/issues/35828
  - fix in the works https://github.com/react-native-community/docker-android/pull/192 & https://github.com/facebook/react-native/pull/35857
- https://github.com/facebook/react-native/issues/35865
  - fix in the works: https://github.com/facebook/react-native/pull/35872
  
#### Local commits to backport to main

1.

**Comment by Titozzz** [comment]
https://github.com/facebook/react-native/commit/38e35df47c59e425fdac34d1f91540b8b8ca2908, not game changer but might ease the migration pain as it could be confusing

**Comment by pmafpinho** [comment]
@kelset this migration is super confusing! I still can't figure out how to update my project on latest version!

**Comment by affansk** [comment]
@kelset happening to me on 0.71.0 https://github.com/facebook/react-native/issues/35838

**Reply by kelset** [reply]
fix in the works here: https://github.com/react-native-community/cli/pull/1798

**Comment by Kudo** [comment]
please help to pick this, thanks! https://github.com/facebook/react-native/commit/a80cf96fc8821dcc07987c9b91dff0c839c8769c

**Comment by kelset** [comment]
other relevant issue that we will want to get fixed asap: https://github.com/facebook/react-native/issues/35828

**Reply by kelset** [reply]
fix in the works here: https://github.com/react-native-community/docker-android/pull/192 + https://github.com/facebook/react-native/pull/35857

**Comment by billnbell** [comment]
we are getting null return on flipper.... See https://github.com/facebook/react-native/issues/35834 any ideas on a fix?

**Reply by cortinico** [reply]
This thread is only for pick requests, and not to highlight bugs.

**Comment by iBotPeaches** [comment]
https://github.com/facebook/react-native/commit/732a02a3c8eef0a9f01f58574aa96f2b47518566 - I would pluck, because it seems to be root of: https://github.com/facebook/react-native/issues/35807, https://github.com/facebook/react-native/issues/35819 and https://github.com/facebook/react-native/issues/35852

**Comment by cortinico** [comment]
I'd like to have this included in .1
- https://github.com/facebook/react-native/pull/35860

This fixes the `--active-arch-only` not working properly for New Architecture on .71.
Not a big deal as prebuilts sort of solved the build speed problem, but it's worth to keep those flags working fine.

This requeires also a bump of the Gradle Plugin

**Comment by jacobp100** [comment]
Could I get https://github.com/facebook/react-native/commit/0c53420a7af306d629350e1244e8e2ccae08a312 ?

**Comment by enahum** [comment]
Can we get this PR https://github.com/facebook/react-native/pull/35869 ? basically fixes a crash while using 0.71 in combination with react-native-navigation

**Comment by Titozzz** [comment]
https://github.com/facebook/react-native/commit/b314e6f147fdfcd71e7ba3b17338bb44356df98e would be nice as it allow to use the new ios template with libs like react-native-firebase https://rnfirebase.io/messaging/usage#background-application-state.

**Comment by kelset** [comment]
another CLI fix https://github.com/react-native-community/cli/pull/1800

**Comment by UNIDY2002** [comment]
I think it would be nice if we include this: https://github.com/react-native-community/cli/pull/1802

**Reply by Titozzz** [reply]
This will be included since we bumped cli to latest!

**Comment by humarkx** [comment]
It would be nice to include a fix for this issue: https://github.com/facebook/react-native/issues/35845
Since the android build fails on 0.71

**Reply by kelset** [reply]
that error report is very incomplete and it's hard to tell if it's a real issue or something wrong in the user configuration. We'll be releasing 0.71.1 shortly with what we have so far since there are already a number of significant fixes ready to go
